From 95455406cb4edbd30a3d92b91740a7aaede63f3d Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 24 Jul 2014 14:22:45 +0000
Subject: [PATCH 0560/1077] MANTIS 0033134 extension Trace DLL: add description
 of trace format

trace.dll, tracehdl.dll, traceNoConfig.dll, tracehdlNoConfig.dll
V1.0.0.6

the applications may use proprietary line formats using writeTraceCooked



git-svn-id: svn://localhost/SelfServiceCommon/trunk@702 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/cpp/Trace/inc/local_def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SelfServiceCommon/Massai/cpp/Trace/inc/local_def.h b/SelfServiceCommon/Massai/cpp/Trace/inc/local_def.h
index 1969decd..c56a7885 100644
--- a/SelfServiceCommon/Massai/cpp/Trace/inc/local_def.h
+++ b/SelfServiceCommon/Massai/cpp/Trace/inc/local_def.h
@@ -21,7 +21,7 @@ trace.dll, tracehdl.dll, traceNoConfig.dll, tracehdlNoConfig.dll
 - line:          "<day>.<month>.<year>\t<hour>:<minute>:<second>,<milliseconds>\t<thread index>\t<level/topic>\t<file>(<line>)"\n
   for "Trace Init" and "Trace Close" the thread id is used; because the thread index cannot be calculated
 - dump:          "D\t\t<dec position>\t<hex position>\t<hex data>\t<data>"
-the applications use proprietary line formats using writeTraceCooked
+the applications may use proprietary line formats using writeTraceCooked
 
 
 \version 1.0.0.5
-- 
2.41.0.windows.1

